p5-travel-app/src/client/js/model.js

```diff
@@ -0,0 +1,146 @@
+import "bootstrap";
+
+const $ = require("jquery");
+import { countdown } from "./form";
+
+const recentTrip = (trip) => {
+  document.querySelector(".caption").style.display = "block";
+  document.querySelector(".caption").style.top = "5vh";
+  $("#tripModal").modal("toggle");
+
+  const tripStart = getTripDate(trip.start);
+  const tripEnd = getTripDate(trip.end);
+  const daysLeft = countdown(new Date(), trip.start);
+  const weather = getWeatherInfo(trip.weatherForecast, daysLeft, tripStart);
+  const div = document.createElement("div");
+  div.classList.add("col-md-4");
+  //section.appendChild(div);
```

Comment:
remove

Reply:
done, clean it in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)